src/plugins/ui_actions/public/service/ui_actions_execution_service.ts

```diff
@@ -84,11 +89,23 @@ export class UiActionsExecutionService {
     setTimeout(() => {
       if (this.pendingTasks.size === 0) {
         const tasks = uniqBy(this.batchingQueue, (t) => t.action.id);
-        if (tasks.length === 1) {
-          this.executeSingleTask(tasks[0]);
-        }
-        if (tasks.length > 1) {
-          this.executeMultipleActions(tasks);
+        if (tasks.length > 0) {
```

Comment:
whats this for ?

Reply:
This is to support `alwaysShowPopup` option above which allows to make sure popup context menu always opens, even if there is just one UI Action to be executed. Otherwise, now the default behaviour is to immediately execute an action if there is only one action. The change allows us to always show the context menu for `ROW_CLICK_TRIGGER`.